sdk/tests/conformance2/reading/format-r11f-g11f-b10f.html

```diff
@@ -0,0 +1,165 @@
+<!--
+
+/*
+** Copyright (c) 2016 The Khronos Group Inc.
+**
+** Permission is hereby granted, free of charge, to any person obtaining a
+** copy of this software and/or associated documentation files (the
+** "Materials"), to deal in the Materials without restriction, including
+** without limitation the rights to use, copy, modify, merge, publish,
+** distribute, sublicense, and/or sell copies of the Materials, and to
+** permit persons to whom the Materials are furnished to do so, subject to
+** the following conditions:
+**
+** The above copyright notice and this permission notice shall be included
+** in all copies or substantial portions of the Materials.
+**
+** THE MATERIALS ARE PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+** EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+** MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+** IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
+** CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+** TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+** MATERIALS OR THE USE OR OTHER DEALINGS IN THE MATERIALS.
+*/
+
+-->
+
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>Test Format R11F_G11F_B10F</title>
+<link rel="stylesheet" href="../../resources/js-test-style.css"/>
+<script src="../../js/js-test-pre.js"></script>
+<script src="../../js/webgl-test-utils.js"></script>
+</head>
+<body>
+<div id="description"></div>
+<div id="console"></div>
+<canvas id="canvas" width="20" height="20"> </canvas>
+<script>
+"use strict";
+description("This tests format R11F_G11F_B10F works as expected");
+debug("MacOSX driver bug. See https://github.com/KhronosGroup/WebGL/issues/1832");
+
+var wtu = WebGLTestUtils;
+var canvas = document.getElementById("canvas");
+var gl = wtu.create3DContext(canvas, undefined, 2);
+
+if (!gl) {
+  testFailed("context does not exist");
+} else {
+  testPassed("context exists");
+  if (gl.getExtension("EXT_color_buffer_float")) {
+    testPassed("Extension EXT_color_buffer_float is available");
+    testRenderbufferReadback(4, 4);
+    testTextureReadback(4, 4);
+  } else {
+    testPassed("Extension EXT_color_buffer_float is unavailable - this is legal");
+  }
+}
+
+function setupColor(testR, testG, testB, value) {
+  var data = new Float32Array(4);
+  data[0] = testR ? value : 0;
+  data[1] = testG ? value : 0;
+  data[2] = testB ? value : 0;
+  data[3] = 1; // Doesn't really matter for RGB formats.
+  return data;
+}
+
+function clearAndVerifyColor(width, height, testR, testG, testB, value) {
+  var data = setupColor(testR, testG, testB, value);
+  // TODO(zmo): are we too tolerant here?
+  var tol = [36, 36, 36];
```

Comment:
The test case fails both on Mac Intel and Linux Nvidia when r/g/b equals to 8191. 

According to the spec of GLES 3.0, take unsigned 11bit floating point number as an example (see spec at https://www.khronos.org/registry/gles/specs/3.0/es_spec_3.0.4.pdf#nameddest=section-2.1.3). The accuracy of different data range will vary. If the data is among 4096 ~ 8191, the exponent would be 27, the accuracy for every one step of mantissa would be 4096/64 = 64. Moreover, the driver will round to  the floor. So all data among 8128 to 8191 would round to 8128 (exponent E is 27, mantissa M is 63). So the deviation for 8191 would be 8191 - 8128 = 63, which is bigger than the tolerant number you set. So the case fails. 

I would like to list some possible biggest deviation for input data for unsigned 11bit floating point number: 
unsigned 11bit floating point number       possbile biggest deviation
     ....                                                             ...
1024 - 2047                                                   15
2048 - 4095                                                   31
4096 - 8191                                                   63
8192 - 16383                                                 127
   ...                                                                  ...

I tried the data 2047, 4095, 8191, 16383 respectively. The result conforms to my deduction.